packages/model-serving/src/components/deploymentWizard/steps/ModelDeploymentStep.tsx

```diff
@@ -16,6 +18,11 @@ export const ModelDeploymentStepContent: React.FC<ModelDeploymentStepProps> = ({
   projectName,
   wizardState,
 }) => {
+  const profileIdentifiers = useProfileIdentifiers(
+    undefined,
+    wizardState.state.hardwareProfileConfig.formData.selectedProfile,
+  );
+
```

Comment:
@nananosirova Does this look right to you? I'm not sure how this work for HW Profiles so I just copied how it was being done in the modal. You can also see where I use them in the `ModelServerTemplateSelectField`